### ISSUE TITLE
saveHTML function renders empty string if not loaded

### ISSUE THREAD

**Issue by emirror-de** [ivopetkov/html5-dom-document-php#27]
Hi there!
I was playing around with the library and found out, that the following snippet (line 267 to 269 of src/HTML5DOMDocument.php) obviously prevents any output if the content has not been loaded by the loadHTML function.
`if (!$this->loaded) { return '<!DOCTYPE html>'; }`

Is there a specific reason that the function terminates with this return statement? Do you know about drawbacks when this snippet is being removed or commented out?

Greetings,
Lewin

**Comment by ivopetkov** [comment]
Hi, thanks for pointing this out. Maybe in the previous versions this code was helpful, but I don't see the benefits now. I'll be happy to learn about your use cases and take a deeper look at the situation.

**Comment by emirror-de** [comment]
Hey, thanks for your reply!
Ok, I integrated it with these lines removed and it is working fine so far.

I am currently working on a PHP Framework that (roughly described) allows users to create modularized websites using the DOMDocument class. The modularization starts already at the file level. There are many other features that are too long to describe here. But as soon as I finished the documentation I will release it under MIT license and if your interested, I will notify you.

Because there are problems with HTML5 support of DOMDocument I was looking for an extension like the one you did.

Greetings,
Lewin
